src/rt/sections_elf_shared.d

```diff
@@ -369,7 +370,13 @@ else
      * Thread local array that contains TLS memory ranges for each
      * library initialized in this thread.
      */
-    @property ref Array!(void[]) _tlsRanges() @nogc nothrow { static Array!(void[]) x; return x; }
+    @property ref Array!(void[]) _tlsRanges() @nogc nothrow {
+        static Array!(void[])* x = null;
+        if (x is null)
+            x = cast(Array!(void[])*).calloc(1, Array!(void[]).sizeof);
+        assert(x);
```

Comment:
I think this is probably too soon to throw Errors, and [safeAssert](https://github.com/dlang/druntime/blob/master/src/rt/util/utility.d#L21) a better fit.

Reply:
Only alternative is to crash or error in `initTLSRanges`. Making the change to `safeAssert`